Add ISBN to book

```diff
@@ -0,0 +1,132 @@
+import java.util.Scanner;
+
+public class Test1 {
+
+   // method to check number is ISBN 
+   public static boolean isISBN(String number) {
+
+      // declare variable
+      int length = 0;
+
+      // remove all hyphens
+      number = number.replace("-", "");
+      // remove all spaces
+      number = number.replace(" ", "");
+
+      // check result string is a number or not
+      try {
+         // except for the case where
+         // ISBN-10 ends with X or x
+         char ch = number.charAt(9);
```

Comment:
I think its better to have magic constants defined under a name a not number literal so that its clear what it is. This might also fail with exception when 9 is out of bounds.